x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="FreeType-based font drawing library for X"
+DESCRIPTION="Xft provides a client-side font API for X applications, \
+making the FreeType font rasterizer available to X clients. Fontconfig \
+is used for font specification resolution. Where available, the RENDER \
+extension handles glyph drawing; otherwise, the core X protocol is used."
+HOMEPAGE="https://www.x.org/"
+COPYRIGHT="2000-2003 Keith Packard"
+LICENSE="libXft"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXft-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f5a3c824761df351ca91827ac221090943ef28b248573486050de89f4bfcdc4c"
+SOURCE_DIR="libXft-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
```

Comment:
You can mark x86_gcc2 as tested, as the recipe builds fine on that arch:
```diff
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

BTW, some contributors like to get rid of the `x86` in `ARCHITECTURES` and of the `x86_gcc2` in `SECONDARY_ARCHITECTURES` but I prefer to keep them if I notice that the recipe builds on these architectures.
FWIW:
I only drop a `x86` (or `?x86`) in `ARCHITECTURES` when the recipe is broken on that architecture.
I only drop a `x86_gcc2` (or `?x86_gcc2`) in ``SECONDARY_ARCHITECTURES`` is when the recipe is broken on that architecture.
And when I find that a recipe builds but I can't test it then I just mark it as untested.

Reply:
Fixed.